Sources/MapboxNavigation/NavigationViewController.swift

```diff
@@ -395,6 +404,19 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
         notifyUserAboutLowVolumeIfNeeded()
     }
     
+    override open func willTransition(to newCollection: UITraitCollection, with coordinator: UIViewControllerTransitionCoordinator) {
+        if #available(iOS 12.0, *) {
```

Comment:
Would it be possible to extract this code into separate method so that if can be called from multiple locations if needed? I think that we also need to add public property that guards this logic (e.g. `usesNightStyleInDarkMode`).